src/components/quickstarts/QuickstartsSidebar.js

```diff
@@ -0,0 +1,193 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import PropTypes from 'prop-types';
+import SuperTilesExperiment from '../../experiments/super_tiles';
+import IOLogo from '../IOLogo';
+import QuickstartFilter from './QuickstartFilter';
+import { Link, Button } from '@newrelic/gatsby-theme-newrelic';
+
+const QuickstartsSidebar = ({
+  isMobile,
+  clearFilters,
+  filters,
+  filtersWithCount,
+  categoriesWithCount,
+  category,
+  handleFilter,
+  handleCategory,
+}) => (
+  <aside
+    data-swiftype-index={false}
+    css={css`
+      grid-area: sidebar;
+      border-right: ${isMobile ? 'none' : '1px solid var(--divider-color)'};
+      height: calc(100vh - var(--global-header-height));
+      position: sticky;
+      top: var(--global-header-height);
+
+      @media screen and (max-width: 760px) {
+        display: block;
+        position: relative;
+        overflow: hidden;
+        width: 100%;
+        height: 100%;
+      }
+    `}
+  >
+    {isMobile && <SuperTilesExperiment isMobile />}
```

Comment:
The `SuperTilesExperiment` has ended, let's add a FIXME comment so we remember to update this when we pull in the latest from `develop`